src/styles/blocks/card.scss

```diff
@@ -0,0 +1,89 @@
+.card {
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  width: 200px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
```

Comment:
`box-sizing: border-box;` is duplicated, so make sense to move it to the `*{ box-sizing: border-box }`